8th-snippets.code-snippets

```diff
@@ -0,0 +1,51 @@
+{
+	// Place your forth-language workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and 
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope 
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is 
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are: 
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. 
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
+	// 	"prefix": "log",
+	// 	"body": [
+	// 		"console.log('$1');",
+	// 		"$2"
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"if then clause": {
+		"scope": "8th",
+		"prefix": "ift",
+		"body": [
+			"${1:condition} if",
+			"\t${2:code}",
+			"then",
+			"$0"
+		]
+	},
+	"if then else clause": {
+		"scope": "8th",
+		"prefix": "ife",
+		"body": [
+			"${1:condition} if",
+			"\t${2:if-code}",
+			"else",
+			"\t${3:else-code}",
+			"then",
+			"$0"
+		]
+	},
+	"word definitiom": {
+		"scope": "8th",
+		"prefix": "def",
+		"body": [
+			"\\ ${1:description}",
+			": ${2:word} \\ ${3:before} -- ${4:after} ",
+			"\t${5:definition} ;",
```

Comment:
I don't know if `\t` in a snippet will expand to "follow the rules" of .editorconfig and the language (e.g. replace tabs with N spaces). If it does, then great. If not, is there a way to do this?

Reply:
Yes, I always have my editor at spaces and use tabs. It works fine.